lib/sampler.js

```diff
@@ -15,10 +15,12 @@
  * limitations under the License.
  */
 
-function foo() {
-    return 'Hello world!';
+class AlwaysSampledSampler {
+    isSampled() {
+        return true;
+    }
 }
 
 module.exports = {
-    foo,
+    AlwaysSampledSampler,
```

Comment:
:+1: There will be probably more samplers, leaving the original one as in the future it is easier to extend.